olive/model/utils/hf_mappings.py

```diff
@@ -73,6 +73,7 @@
 # there is the priority order: NUM_HEADS_NAMES[0] and HIDDEN_SIZE_NAMES[0] are the first choice
 # which means user can override the value in config file
 NUM_HEADS_NAMES = ("num_heads", "num_attention_heads", "n_head", "n_heads", "encoder_attention_heads")
+NUM_KEY_VALUE_HEADS_NAMES = "num_key_value_heads"
```

Comment:
Can you make this a tuple like `NUM_HEADS_NAMES`? Line 261 in transformers optimization file loops over it but currently it a string.

Reply:
I think I made it a tuple before but forgot the trailing `,`, so it probably got removed by the formatter as redundant parentheses. Fixed.